### PR TITLE
Docs: Add missing parameter for Toast

### PR DESCRIPTION
**Pull request by Neet-Nestor**
* Docs: The parameter `color` exists in the code but not in the docs. Add the parameter and a corresponding example for it in Toast.

image
image

### CODE REVIEW

File: docs/src/Toast.doc.js [hunk 1]
```diff
@@ -97,6 +102,56 @@ function ToastExample() {
...
+          }}
+          paddingX={1}
+          position='fixed'
+        >
+          {showToast && (
+            <Toast
+              color="red"
+              text={
+                <>
+                  Saved to{' '}
```
Comment (christianvuerings) [inline docs/src/Toast.doc.js:137]: @Neet-Nestor Could we update this to be an error message instead? E.g. `Oops, we couldn't find that!`. It makes the intent more clear

**Comment by christianvuerings** [comment]
See https://github.com/pinterest/gestalt/pull/760  for more information on why it's no longer documented. I'm ok adding it back in since it's confusing for developers.

**Comment by Neet-Nestor** [comment]
Thanks for the advise! Updated:
image

**Comment by Neet-Nestor** [comment]
> See #760 for more information on why it's no longer documented. I'm ok adding it back in since it's confusing for developers.

Didn't know it. I'll also check with our designer about this. Thank you!

**Comment by christianvuerings** [comment]
@Neet-Nestor merged - thanks for the fix!

**Pull request merged**